examples/annotation_import/conversational.ipynb

```diff
@@ -1,389 +1,732 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    )
</pre><p>Make sure to get rid of unused imports</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>